shell/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint ."
+    "lint": "eslint . && tsc"
```

Comment:
Your commit message claims you specify `--noEmit` here but I don't see it.  Am I missing something?

Separately: maybe put the `tsc` check before the `eslint` check; I anticipate it running faster.

Reply:
Re: noEmit, it's in the tsconfig.json file, which I apparently forgot to `git add`, oops. It's there now.

Re: ordering, I've observed the opposite: eslint is noticeably faster.

Reply:
If I understand correctly, eslint as configured right now is running very few checks on the codebase, which might explain the speed.  Happy to leave this as-is until there's an inflection point. :)

In due time, we'll probably want to enable a broader suite of eslint checks, like `extends: ["airbnb-base", "plugin:meteor/recommend", "plugin:import/errors", "plugin:import/warnings"]` to the eslintrc, which will do substantially more work/checks.  Doing so right now, however, will give us on the order of several tens of thousands of things to fix, and some of the "safe" automated changes (`--fix`) appear to break things in ways I haven't had a chance to investigate in full.  We can take this slowly.

Reply:
Yeah, that was my thinking: get it in place first and add checks incrementally.